### ISSUE TITLE
Newly joined chats not loading in landscape

### ISSUE THREAD

**Issue by mursicale** [urbit/urbit#3474]
**Describe the bug**
Joining a chat shows only the oldest message, or nothing at all. 
Posting to chat succeeds, but does not display in landscape.

Able to see chat history in the shell.

**To Reproduce**
1. Join any chat from a group (this has happened in groups I was already in as well as newly joined group
2. View chat

**Expected behavior**
Expected to see full chat log

**Desktop (please complete the following information):**
 - OS: Linux 5.7.0-1-amd64 #1 SMP Debian 5.7.6-1 (2020-06-24) x86_64 GNU/Linux
 - Browser Firefox 68.10.0esr (64-bit)
 - %base-hash 0vu.k9jrq.meajk.1a6i2.q39sd.l4rbi.37h5i.f3j5f.5jujs.lg7ku.e135b

**Additional context**
Might this be related to the scrolling bug [here](https://github.com/urbit/urbit/issues/3465)?

After bringing the ship down and back up, chats appear as normal.

**Comment by matildepark** [comment]
Looks like a separate bug related to the first time we join a chat and get the backlog. I would imagine refreshing would alleviate it too. @tylershuster as you're doing a bug smash PR are you aware of this one?

**Comment by tylershuster** [comment]
I've experienced similar behavior but nothing this dramatic. I'll add it to the list.

**Comment by mursicale** [comment]
@matildepark you're totally correct: refreshing the page works, it's my tendency to catastrophize that overlooked the simple way.

**Comment by tylershuster** [comment]
Ok, this is due to `mailboxSize` (derived in `chat/app.tsx`) which means that you can get different results by navigating out of the app and back in, once `mailboxSize` gets recalculated. However, it still doesn't work because only the latest 20 chats are loaded.

**Comment by matildepark** [comment]
Closed in #3476.

**Comment by tylershuster** [comment]
@matildepark this is still at large

**Comment by matildepark** [comment]
I didn't encounter it when I left and rejoined a chat, so ... weird.

**Comment by tylershuster** [comment]
Working on some other chat navigation fixes and noticed it. I'll probably PR a fix soon

**Comment by tylershuster** [comment]
@mursicale I would be interested to know if you encounter this after the most recent update. I tried to reproduce and while it did take a long time to load, it at least told me that it was loading the whole time.

**Comment by mursicale** [comment]
I checked this morning, and the issue is only slightly different. In fact, I found that all the chats from a couple of groups were gone, and I had to rejoin them. 

I cleared browser cache, rebooted my ship, and confirmed that I'm on the latest base hash (`8q5mn`). 
When I did rejoin chats, they started from the top, and one or two started scrolling after a bit, but others did not. When I posted in one chat, I was taken to the top, but this did not happen in another. Some chats went back to the top after I navigated away, others stayed put.
